### ISSUE TITLE
Cardbox System

### ISSUE THREAD

**Issue by obstschale** [RubenGees/Vocabletrainer#4]
I want to use a vocabulary app for learning my spanish vocs. However I'm a fan of the cardbox system. So the app presents me the appropriated words to repeat. I now there I can select in a limited way which vocabularies to learn (new ones, hard to remeber, etc.) but the actual cardbox system (like in phase 6) is much more powerful and proved to be a good way to learn. 

Did you plan to implement this feature someday?

**Comment by rubengees** [comment]
Yes I planned to do that for a long time, but I currently have not the time to implement a whole new mode...

**Comment by rubengees** [comment]
I thought of adding an "intelligent" mode which basically select those vocables which have the worst correct/wrong rate and which are most seldom learned. Is this somehow what you had in mind?

**Comment by obstschale** [comment]
This sounds also nice, but it is not really the same. My idea (actual it's not my idea but a proven scientific concept) is that vocables are presented regularly - don't matter if you knew it before or it was wrong. Take at look at phase-6. They cite some papers, which explain exactly that concept: http://www.phase-6.de/service/wissenschaftlicher_hintergrund.html

**Comment by rubengees** [comment]
Okay, thank you for pointing that out. 
I think that would not go well with the concept of the current Vocabletrainer? 
Now you have one test and have to finish it before you get results. A test with the cardbox system would take very long and only few users would complete it in one run I think. Thus I was thinking about the intelligent mode, which somehow does what the cardbox system does, but without forcing the user to stay on the test.
I hope you get what I am trying to say :D
What do you think?

**Comment by obstschale** [comment]
Sounds also good. :+1:
